### ISSUE TITLE
🐞 Bug Report: Elementor Post Widget Accessibility aria-label still missing from Thumbnail link

### ISSUE THREAD

**Issue by srdesigns** [elementor/elementor#21526]
Referencing an issue that was half resolved in the 3.1.1 update. While  the aria-label was added to the Read More link, it is still missing from the Thumbnail Link, causing the "Links do not have a discernible name" error.
___ 

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro **v3.11**! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂

_Originally posted by @nicholaszein in https://github.com/elementor/elementor/issues/13784#issuecomment-1428364853_

**Comment by nicholaszein** [comment]
Hello @srdesigns! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards
